### PR TITLE
[Edit] Update HTML style

### PR DESCRIPTION
**Pull request by kkkeevvvin**
### Description

In the [style](https://www.codecademy.com/resources/docs/html/attributes/style) page, it only mention that it's better using external CSS style sheet. But, I think it also important to mention how-to.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@kkkeevvvin you titled this PR as if it was dealing with an existing issue, but it doesn't actually have anything to do with that issue, so I changed the name to avoid further confusion.

File: content/html/concepts/attributes/terms/style/style.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The **`style`** attribute specifies how an element will render in a web browser,
 When the `style` attribute is used inline, it overrides any styles set globally.
 
 > **Note:** The styling of a page or elements on the page can also be embedded in the `<head>` element using the [`<style>`](https://www.codecademy.com/resources/docs/html/elements/style) tag. However, most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet. This helps to keep the code clean and is more easily maintained.
+> The method for using external CSS style sheet can be seen at [`<link>`](https://www.codecademy.com/resources/docs/html/elements/link).
```
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/style/style.md:22]:
```suggestion
> The method for using an external CSS style sheet can be seen at [`<link>`](https://www.codecademy.com/resources/docs/html/elements/link).
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kkkeevvvin I did a first review and left a comment.

**Review by caupolicandiaz — APPROVED** [review]
@kkkeevvvin @SSwiniarski Just updated the language in the note to condense it a bit

**Pull request merged**